docs/README.md

```diff
@@ -0,0 +1,23 @@
+# SimManager Documentation
```

Comment:
Did you mean `MaestroWF` instead of `SimManager` throughout this document?

Reply:
The full name is `Maestro Workflow Conductor` -- if that should be the official name.